routes/register.js

```diff
@@ -1,7 +1,7 @@
 const express = require('express')
 const router = express.Router()
 
-const { register } = require('../services/auth')
+const { register } = require('../controllers/register')
```

Comment:
```suggestion
const { post } = require('../controllers/register')
```
You are exporting `post` from the controller. Thats what should be used here